README.md

```diff
@@ -1,27 +1,57 @@
 # mathjax-api
-A Node.js HTTP API to render MathJax equations in HTML content
 
-This service uses the MathJax-node implementation that is optimised for mathjax embedded in HTML content. It is currently limited to LaTex to SVG conversion, but feel free to extend it :)
+A Node.js HTTP API to render MathJax equations in HTML content.
 
-Defaults port 8080.
+This service uses the MathJax-node implementation that is optimised for mathjax embedded in HTML content. It is currently limited to Tex to SVG conversion.
 
-Node usage: 
-```
-$ npm install
-$ node app.js
-```
+## Getting started
 
-Test the server is working by doing a HTTP GET or navigate to http://localhost:8080
+### Prerequisites
 
-```
-$ curl http://localhost:8080
-```
+To run the API locally you must have the following:
 
-To post content to the service for conversion, put your content in the "input" field using a application/x-www-form-urlencoded format:
+1. [nvm](https://github.com/nvm-sh/nvm) installed:
 
+   ```shell
+   brew install nvm
+   ```
+
+   :warning: Make sure to follow the instructions provided at the end of the install to configure up your shell profile. Once you have done this, you will need to reload your shell for this to take effect.
+
+2. The node version specified in [`.nvmrc`](./.nvmrc) installed through nvm:
+
+   ```shell
+   nvm install
+   ```
+
+### Build and run
+
+Once you have node installed, run the following to install the dependencies and run the server:
+
+```shell
+make debug
 ```
```

Comment:
indentation.

Reply:
This ins't a numbered list so I believe the indentation is correct?